### DISCUSSION TITLE
Project Naming Conventions

### DISCUSSION THREAD

**Discussion by iamadamreed** [adamreed90/dapr-samples#3] [Ideas]
I had **seemed** to have issues with `docker-compose` using "." (dots) in the Project names (ex: SomeProject.Data) so I believe we should try to avoid that or get some more clarification from the core developers or others more knowledgeable. 

I currently scaffolded:
Solution: HelloDotNet6
Project: Console

**Comment by georgestevens99** [comment]
I am OK with avoiding problems, including avoiding dots in names, so as to get a basic running example.  Then we can build on that if neccessary.
